assets/src/edit-story/masks/index.js

```diff
@@ -97,9 +193,175 @@ export const MASKS = [
     ratio: 1.15473441108545,
   },
   {
-    type: MaskTypes.BLOB,
-    name: __('Blob', 'web-stories'),
-    path: CLIP_PATHS[MaskTypes.BLOB],
+    type: MaskTypes.BLOB_1,
+    name: __('Blob 1', 'web-stories'),
+    path: CLIP_PATHS[MaskTypes.BLOB_1],
+    ratio: 1,
+  },
+  {
+    type: MaskTypes.BLOB_2,
+    name: __('Blob 2', 'web-stories'),
+    path: CLIP_PATHS[MaskTypes.BLOB_2],
+    ratio: 1,
+  },
+  {
+    type: MaskTypes.BLOB_3,
+    name: __('Blob 3', 'web-stories'),
+    path: CLIP_PATHS[MaskTypes.BLOB_3],
+    ratio: 1,
+  },
+  {
+    type: MaskTypes.BLOB_4,
+    name: __('Blob 4', 'web-stories'),
+    path: CLIP_PATHS[MaskTypes.BLOB_4],
+    ratio: 1,
+  },
+  {
+    type: MaskTypes.BLOB_5,
+    name: __('Blob 5', 'web-stories'),
+    path: CLIP_PATHS[MaskTypes.BLOB_5],
+    ratio: 1,
+  },
+  {
+    type: MaskTypes.BLOB_6,
+    name: __('Blob 6', 'web-stories'),
+    path: CLIP_PATHS[MaskTypes.BLOB_6],
+    ratio: 1,
+  },
+  {
+    type: MaskTypes.BLOB_7,
+    name: __('Blob 7', 'web-stories'),
+    path: CLIP_PATHS[MaskTypes.BLOB_7],
+    ratio: 1,
+  },
+  {
+    type: MaskTypes.BLOB_8,
+    name: __('Blob 8', 'web-stories'),
+    path: CLIP_PATHS[MaskTypes.BLOB_8],
+    ratio: 1,
+  },
+  {
+    type: MaskTypes.BLOB_9,
+    name: __('Blob 9', 'web-stories'),
+    path: CLIP_PATHS[MaskTypes.BLOB_9],
+    iconPath: ICON_CLIP_PATHS[MaskTypes.BLOB_9],
+    ratio: 653.02 / 836.92,
+    iconRatio: 1,
+  },
+  {
+    type: MaskTypes.GRID_2,
+    name: __('Grid 2', 'web-stories'),
+    path: CLIP_PATHS[MaskTypes.GRID_2],
+    iconPath: ICON_CLIP_PATHS[MaskTypes.GRID_2],
+    ratio: 362 / 619,
+    iconRatio: 1,
+  },
+  {
+    type: MaskTypes.GRID_3,
+    name: __('Grid 3', 'web-stories'),
+    path: CLIP_PATHS[MaskTypes.GRID_3],
+    iconPath: ICON_CLIP_PATHS[MaskTypes.GRID_3],
+    ratio: 360.8 / 619,
+    iconRatio: 1,
```

Comment:
I can't quite wrap my head around this: why is there a different `iconRatio`? Are we showing a skewed icon?

Reply:
No we're not showing skewed icons, but we have a separate icon ratio for icons we wanna draw relative to a `0-1` square. this allows us to visually balance the icons better while still having the paths in the editor be hugged by the transform box.

Again, I think this is a temporary work around until we can play with this more after beta. With complex shapes it makes sense that they need to altered to be visible & look nice as icons. We can come up with a more elegant way to do this down the line that alters this component more drastically after beta, but this is quick and fits within our existing system for now.